app/views/layouts/find_layout.html.erb

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+<head>
+  <title><%= content_for(:page_title) %> <%= t("service_name.find") %> - GOV.UK</title>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/favicon.ico" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-mask-icon.svg", rel: "mask-icon", type: "image/svg", color: "#0b0c0c" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon.png", rel: "apple-touch-icon", type: "image/png" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-152x152.png", rel: "apple-touch-icon", type: "image/png", size: "152x152" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-167x167.png", rel: "apple-touch-icon", type: "image/png", size: "167x167" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-180x180.png", rel: "apple-touch-icon", type: "image/png", size: "180x180" %>
+  <%= stylesheet_link_tag "accessible-autocomplete/dist/accessible-autocomplete.min" %>
+  <%= stylesheet_link_tag "application" %>
+  <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
We should generate `find` version for less bloat

Reply:
This is done. We paired on it.